src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -288,11 +289,9 @@
                              [render-field child :idx idx])
                            children)
                         (when (< idx (dec row-count))
-                          [^{:key (str "spacer-" row-count)}
-                            [:div.application__question-group-spacer]])))
+                          (group-spacer idx))))
```

Comment:
Should a new component just be created here instead:  `[group-spacer idx]`.

Reply:
Indeed, still gotta figure out re-frame :)

Reply:
Yeah works:+1: